test/parallel/test-crypto-key-objects.js

```diff
@@ -64,6 +64,16 @@ const privateDsa = fixtures.readKey('dsa_private_encrypted_1025.pem',
   });
 }
 
+{
+  assert.throws(() => KeyObject.from('invalid_key'), {
+    name: 'TypeError',
+    code: 'ERR_INVALID_ARG_TYPE',
+    message:
+      'The "key" argument must be an instance of CryptoKey. Received type ' +
```

Comment:
```suggestion
      'The "key" argument must be of type CryptoKey. Received type ' +
```

Reply:
I fixed it. 
Thank you very much. It is my first contribution in Node core.